src/clients/catalog.ts

```diff
@@ -28,4 +47,85 @@ export class Catalog extends JanusClient {
       tracing: createTracing(metric, tracingConfig),
     })
   }
+
+  public createSeller(
+    {
+      SellerId,
+      Name,
+      Email,
+      Description = '',
+      ExchangeReturnPolicy = '',
+      DeliveryPolicy = '',
+      UseHybridPaymentOptions,
+      UserName = '',
+      Password = '',
+      SecurityPrivacyPolicy = '',
+      CNPJ = '',
+      CSCIdentification,
+      ArchiveId = '',
+      UrlLogo = '',
```

Comment:
I'd extract this `defaultSeller` on a util function (maybe) to keep the method's signature lean